src/utils/hash/index.ts

```diff
@@ -2,7 +2,7 @@
  * Hashes Exports
  */
 export * as poseidon from '@noble/curves/abstract/poseidon';
-export * from '../selector'; // Preserve legacy export structure
+export * from './selector'; // Preserve legacy export structure
```

Comment:
cc @tabaktoni @penovicp this was an only open question, should be ok to move this inside /hash folder?